### PR TITLE
ARTEMIS-4093: Expose properties and support resource adapter in J2EE environments

### PR DESCRIPTION
**Pull request by hawkdshrh**
Fixes to expose properties in environments not recognizing is* accessors.

### CODE REVIEW

**Comment by clebertsuconic** [comment]
you should rebase this, and fix the title of your commit. you amend the wrong commit..

**Comment by clebertsuconic** [comment]
I converted your commit to draft until you fix this.

File: artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ActiveMQResourceAdapter.java [hunk 1]
```diff
@@ -821,6 +821,10 @@ public void setUseTopologyForLoadBalancing(Boolean useTopologyForLoadBalancing)
       raProperties.setUseTopologyForLoadBalancing(useTopologyForLoadBalancing);
    }
 
+   public Boolean getUseTopologyForLoadBalancing() {
+      return raProperties.isUseTopologyForLoadBalancing();
```
Comment (clebertsuconic) [inline artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ActiveMQResourceAdapter.java:825]: why?


isUseTopology is the standard for Java Beans.

Why do you need the get method?

File: artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java [hunk 2]
```diff
@@ -619,6 +649,10 @@ public void setUseLocalTx(final Boolean localTx) {
       this.localTx = localTx;
    }
 
+   public Boolean getRebalanceConnections() {
```
Comment (clebertsuconic) [inline artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java:652]: why? same question as before.

File: artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ActiveMQResourceAdapter.java (hunk 1 above)
Comment (hawkdshrh) [reply artemis-ra/src/main/java/org/apache/activemq/artemis/ra/ActiveMQResourceAdapter.java:825]: Unfortunately, this is not recognized in some environments.  Websphere Application Server 8.x is one of these.

File: artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java (hunk 2 above)
Comment (hawkdshrh) [reply artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java:652]: Unfortunately, these methods are not recognized in some environments, causing failure to expose the properties (Websphere Application Server is one such).

**Pull request merged**